providers/desktop_background.rb

```diff
@@ -11,9 +11,10 @@
 
 action :setup do
   begin
-    package "dconf-tools" do
```

Comment:
I should not remove this resource, just because the package is included in the distro.